### DISCUSSION TITLE
Base code - Where is it located, and how can it be opened?

### DISCUSSION THREAD

**Discussion by vivianjoshua** [stella-cv/stella_vslam#499] [Q&A]
Hello,
Please pardon me if this is a stupid question. I am new to SLAM and am exploring stuff.
I have installed stella_vslam. Is it possible to view / change the base code? If yes, where can I find this?
My folder structure looks like this:
![image](https://user-images.githubusercontent.com/79590654/235729879-b44b1c4e-615d-4169-bc43-356a0cb92bf2.png)
@shinsumicco @MikiyaShibuya @kensakurada

**Comment by Dylan-Powers** [comment]
I am far from an expert, but I believe the source code exists in the `src` directory.
